syntax = "proto3";

package Protocol;
message LOGIN_REQ {
  string email = 1;
  string password= 2;
}